crates/core_simd/src/swizzle.rs

```diff
@@ -214,6 +214,16 @@ where
     /// Rotates the vector such that the first `OFFSET` elements of the slice move to the end
     /// while the last `self.len() - OFFSET` elements move to the front. After calling `rotate_elements_left`,
     /// the element previously at index `OFFSET` will become the first element in the slice.
+    /// ```
+    /// # #![feature(portable_simd)]
+    /// # use core::simd::Simd;
```

Comment:
You probably need to reference the actual crate like this:
https://github.com/rust-lang/portable-simd/blob/abd5d05ec4de9eb11fd3dfd84cfc1c6e75fb5eba/crates/core_simd/src/vector.rs#L122-L123

Reply:
yup, that did it, thanks!